ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -310,6 +251,20 @@ + (void)configureDocumentViewController:(PTDocumentViewController*)documentViewC
     [(PTFlutterViewController *)documentViewController applyViewerSettings];
 }
 
++ (id)getConfigValue:(NSDictionary*)configDict configKey:(NSString*)configKey class:(Class)class error:(NSError**)error
+{
+    id configResult = configDict[configKey];
+
+    if (![configResult isKindOfClass:[NSNull class]]) {
+        if (![configResult isKindOfClass:class]) {
+            NSString* errorString = [NSString stringWithFormat:@"config %@ is not in expected %@ format.", configKey, class];
+            *error = [NSError errorWithDomain:@"com.flutter.pdftron" code:NSFormattingError userInfo:@{@"message": errorString}];
```

Comment:
Instead of `@"message"` as the key, you should use [NSLocalizedDescriptionKey](https://developer.apple.com/documentation/foundation/nslocalizeddescriptionkey?language=objc) (so that it shows up via a standard property, https://developer.apple.com/documentation/foundation/nserror/1414418-localizeddescription?language=objc)